ui/src/components/Forbidden/Forbidden.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Grid, Button, Icon } from 'semantic-ui-react';
+
+export class Forbidden extends Component {
+  render() {
+    return (
+      <div className="frontsite">
```

Comment:
Is `frontsite` used? If it is not replacing the whole frontsite I find it a bit generic, maybe we can use something more specific for example `fs-forbidden` or just `forbidden` nested inside the proper `frontsite`! ping @kprzerwa!